packages/shared/src/components/modals/streaks/FirstStreakModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from 'react';
+import { LazyModalCommonProps, Modal } from '../common/Modal';
+import { DayStreak, Streak } from '../../streak/popup';
+import { IconSize } from '../../Icon';
+import { Button, ButtonVariant } from '../../buttons/ButtonV2';
+import CloseButton from '../../CloseButton';
+
+export default function FirstStreakModal({
+  onRequestClose,
+  ...props
+}: LazyModalCommonProps): ReactElement {
+  return (
+    <Modal
+      {...props}
+      onRequestClose={onRequestClose}
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.XSmall}
+      className="p-6"
```

Comment:
This seems odd, shouldn't it just maintain modal guidelines?

Reply:
There was actually no padding probably because I did not use the Modal.Body. Let me check if I should actually be using them.